widgetssdk/src/main/java/com/glia/widgets/callvisualizer/ActivityWatcherForDialogs.kt

```diff
@@ -304,6 +326,7 @@ class ActivityWatcherForDialogs(
     }
 
     private fun showVisitorCodeDialog(resumedActivity: WeakReference<Activity?>) {
+        if (!Glia.isInitialized()) return
```

Comment:
I think it should at least log an error that SDK is not initialized. Also, I think after Karl changes in PR#487 it would throw an exception "companyName not set during GliaWidgets.init..." if SDK has not been initialised

Reply:
@gugalo Okay, I rolled this change back.

And I decided to hide the "Show visitor code" button until "Init Glia Widgets" is tapped. This will prevent example app from crash.

So, to see the "Show visitor code" button we need to tap the "Init Glia Widgets" button. I believe it's more friendly UX.

Here it is: 6dac5d216e2d4158073098d4d01c236dfa191a58